app/backend/src/couchers/email/__init__.py

```diff
@@ -37,5 +37,5 @@ def send_email(sender_name, sender_email, recipient, subject, plain, html):
 def send_email_template(recipient, subject, template_file, template_args={}):
     plain, html = _render_email(subject, template_file, template_args)
     return send_email(
-        config["NOTIFICATION_EMAIL_SENDER"], config["NOTIFICATION_EMAIL_ADDRESS"], recipient, subject, plain, html
+        config.NOTIFICATION_EMAIL_SENDER, config["NOTIFICATION_EMAIL_ADDRESS"], recipient, subject, plain, html
```

Comment:
Otherwise LGTM but this line is failing tests with `AttributeError: 'dict' object has no attribute 'NOTIFICATION_EMAIL_SENDER'`

Reply:
Thanks; I really need to get my pgsql test environment up.